### PR TITLE
[Freifunk] remove *.freifunk.net, more working hosts

### PR DESCRIPTION
**Pull request by julianladisch**

### CODE REVIEW

**Comment by numismatika** [comment]
maybe you should secure the board cookie too
```
	<securecookie host="forum\.freifunk\.net$" name=".+" />
```

File: src/chrome/content/rules/Freifunk.xml [hunk 1]
```diff
@@ -33,50 +28,71 @@
...
+	<target host=            "pad.freifunk.net"/>
+	<target host=        "twitter.freifunk.net"/>
+	<target host=          "vimeo.freifunk.net"/>
+	<target host=           "wiki.freifunk.net"/>
+	<target host=            "www.freifunk.net"/>
+	<target host=        "youtube.freifunk.net"/>
+
+	<securecookie host="^(bug|forum|blog\.guetersloh|pad)\.freifunk\.net$" name=".+" />
+
+	<exclusion pattern="^http://(www\.)?freifunk\.net/+(?!favicon\.ico|wp-content/|wp-includes/)" />
```
Comment (TheNavigat) [inline src/chrome/content/rules/Freifunk.xml]: I can't see why that exclusion pattern exists. The site works perfectly on HTTPS. I think it was probably broken by the time the PR was created.

**Comment by semenko** [comment]
Looks like this has a conflict -- happy to merge if it's resolved. :fireworks:

**Comment by semenko** [comment]
Awesome -- thanks!

**Pull request merged**